blog/src/Blog/Post/PostController.php

```diff
@@ -48,15 +49,13 @@ public function add(Request $request, ValidatorInterface $validator): Response
 
         if ($request->getMethod() === Method::POST) {
             $form = new PostForm();
-            if ($form->load($parameters['body']) && $validator
-                    ->validate($form)
-                    ->isValid()) {
+            if ($formHydrator->populate($form, $parameters['body']) && $form->isValid()) {
                 $this->postService->savePost($this->userService->getUser(), new Post(), $form);
 
                 return $this->webService->getRedirectResponse('blog/index');
             }
 
-            $parameters['errors'] = $form->getFormErrors();
+            $parameters['errors'] = $form->getValidationResult()->getErrorMessagesIndexedByAttribute();
```

Comment:
Looks more complicated.

Reply:
There are several cases of errors usage:
- simple list of errors
- errors indexed by attribute
- errors indexed by path

All cases implemented in `Result` object. Should we duplicate this methods in form class ?